ministryofjustice/laa-review-criminal-legal-aid
# Crimre 188 add new user

## Description of change

Adds ability for a user to be added to the system

Will raise a further PR that deals with the PG::UniqueViolation error thats thrown so that it is rendered to the page on the form.

## Link to relevant ticket
[CRIMRE-188](https://dsdmoj.atlassian.net/browse/CRIMRE-188)
## Notes for reviewer

## Screenshots of changes (if applicable)

### After changes:

Form:
<img width="1024" alt="Screenshot 2023-03-07 at 15 35 49" src="https://user-images.githubusercontent.com/13377553/223471889-50522d05-2138-47c2-94af-71f15721aa3d.png">


Errors:
<img width="1147" alt="Screenshot 2023-03-07 at 15 36 43" src="https://user-images.githubusercontent.com/13377553/223471860-339a1a1c-5093-47d0-bb10-9edb64cdecb4.png">


## How to manually test the feature
Sign in as user with admin access

Add a new user by email

Validations:
- try adding an email that is not properly formatted and you should receive an error
- try adding a user without adding an email and you should also receive an error

[CRIMRE-188]: https://dsdmoj.atlassian.net/browse/CRIMRE-188?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
